app/services/open_library_service.rb

```diff
@@ -0,0 +1,7 @@
+class OpenLibraryService
+  def execute(bibkeys)
+    s = "https://openlibrary.org/api/books?bibkeys=#{bibkeys}&format=json&jscmd=data"
```

Comment:
Here you can define the https://openlibrary.org/api/ as a class variable in the class OpenLibraryService or like an environment variable using the secrets. 
If you add a new service that use part of the same url you can use it without declaring the same variable again.

Reply:
Done